CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#[[
+This source file is part of the DocC open source project
+
+Copyright (c) 2024 Apple Inc. and the DocC project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+#]]
+
+cmake_minimum_required(VERSION 3.24)
+
+project(LMDB
+  LANGUAGES C)
+
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
```

Comment:
Mostly laziness, but that's easy to adopt.

Reply:
Wait, why would we use `GNUInstallDirs` here? This is the _build_ tree, not the install tree.

Reply:
oh yeah, fair enough. Do these need to go in special directories then. The build directory locations should all be set in the LMDB config file.

Reply:
I still feel a tad weird about projects poking directly into another project's build directory (unless they're using the `*Config.cmake` file to get the appropriate locations and layouts), but I'll accept it for now since the rest of the Swift ecosystem does this.